package.json

```diff
@@ -1307,6 +1307,11 @@
                         ]
                     }
                 },
+                "docker.networks.hideBuiltInNetworks": {
```

Comment:
Nit: Generally we try to use the positive instead of the negative, i.e. instead of `hideBuiltInNetworks`, default `false`, it would be `showBuiltInNetworks`, default `true`. Negatives can get confusing when they are false (i.e. it's a double-negative).